quantize.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+
+"""Script to execute quantization on a given model."""
+
+import subprocess
+import argparse
+import sys
+import os
+
+
+def main():
+    """Parse the command line arguments and execute the script."""
+
+    parser = argparse.ArgumentParser(
+        prog='Quantization Script',
+        description='This script quantizes a model.'
+    )
+
+    parser.add_argument("models", nargs='+', dest='models')
+    parser.add_argument(
+        '-r', '--remove-16', action='store_true', dest='remove_f16',
+        help='Remove the f16 model after quantizing it.'
+    )
+
+    args = parser.parse_args()
+
+    for model in args.models:
+
+        model_path = os.path.join("models", model, "ggml-model-f16.bin")
```

Comment:
looks like `model_path` is a file, but then you're doing `os.listdir()` on a file? not sure how that would work.

also we would want to limit the directory listing to just a glob of the proper matching filenames since there's many other things in these directories.

Reply:
I think I missunderstood the original script (`for i in `ls models/$1/ggml-model-f16.bin*`), looks like it is in fact a file. Thank you for the observation.